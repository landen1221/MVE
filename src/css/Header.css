.Header h2 {
  /* padding-top: 10px; */
  margin-right: 10%;
  margin-left: 10%;
  font-size: calc(16px + 6 * ((100vw - 320px) / 680));
}
