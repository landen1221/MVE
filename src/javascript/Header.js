import ButtonSection from "./ButtonSection";
import "../css/Header.css";

const Header = ({ vaccines }) => {
  return (
    <div className="Header">
      <h2>
        Real stories from real people to help you make an informed decision!
      </h2>
      <ButtonSection vaccines={vaccines} />
    </div>
  );
};

export default Header;
