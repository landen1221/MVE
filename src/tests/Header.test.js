import ReactDOM from "react-dom";
import Header from "../javascript/Header";
import { MemoryRouter } from "react-router";
import { vaccines } from "./globalTestVariables";
import { render } from "@testing-library/react";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <MemoryRouter>
      <Header vaccines={vaccines}></Header>
    </MemoryRouter>,
    div
  );
});

it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <Header vaccines={vaccines}></Header>
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
