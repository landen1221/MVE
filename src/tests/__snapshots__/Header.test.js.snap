// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Header"
  >
    <h2>
      Real stories from real people to help you make an informed decision!
    </h2>
    <div
      id="ButtonSection"
    >
      <a
        href="/covid"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          id="VaccineButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            COVID
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </a>
      <a
        href="/vaccine/johnsonandjohnson"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          id="VaccineButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Johnson & Johnson
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </a>
      <a
        href="/vaccine/astrazeneca"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          id="VaccineButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            AstraZeneca
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </a>
      <a
        href="/vaccine/pfizer"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          id="VaccineButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Pfizer
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </a>
      <a
        href="/vaccine/moderna"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          id="VaccineButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Moderna
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
